### ISSUE TITLE
Renamed letsencrypt-zimbra script

### ISSUE THREAD

**Issue by VojtechMyslivec** [VojtechMyslivec/letsencrypt-zimbra#80]
Hi, I also installed snap 1.20 and the last script for getting the certificates, but I had to rename the script from `obtain-and-deploy-letsencrypt-cert.sh` to `letsencrypt-zimbra.sh`
I also symlinked the old name to the new one, but of course it didn't work.

Every night the certbot runs and fails like this:
```
obtain-and-deploy-letsencrypt-cert.sh: warning: You are using deprecated script name, change it to 'letsencrypt-zimbra.sh'
```

I have tried to find the cron that runs it, but somehow there is no cron line.
Any ideas?
Thanks!

_Originally posted by @dhcmega in https://github.com/VojtechMyslivec/letsencrypt-zimbra/issues/74#issuecomment-998422989_

**Comment by VojtechMyslivec** [comment]
Hi @dhcmega 

it would be nice to open a new issue next time as your problem seems not to be relevant to exchange of root certificates.

As you can notice, the log message you refer is just a _warning_. The script does not fail on this probably. Are there some other error messages? What exactly does not work?

Could you check (and paste here) your cron file? In the README, I recommend a file placed in `/etc/cron.d/` but there are several location where a cron job can be defined: `/etc/crontab`, `/etc/cron.*/`,  or `/var/spool/cron/crontabs/`.

I also don't understand what did you really "renamed". If you are using `letsencrypt-zimbra` from the git repo, you should not rename or move anything but the configuration file described in the README.

Vojtech

**Comment by dhcmega** [comment]
Hi @VojtechMyslivec yes sorry, you are right.
I thought it was related as this problem was the result of solving the root cert.
I changed it's name because of this error:
```
obtain-and-deploy-letsencrypt-cert.sh: warning: You are using deprecated script name, change it to 'letsencrypt-zimbra.sh'
```

I just ran it again and it turns out is not working:
```
letsencrypt-zimbra.sh: error: The root CA certificate '/opt/letsencrypt-zimbra/root_certs/ISRG_Root_X1.crt' isn't readable file.
```

I don't how I made it work when I was able to manually regenerate the certs.

I'm so done with zimbra, I will just create a new server with webmin and create a simple mail server, zimbra is way too complicated, etc

Thank you!

PS: The cron was located at /etc/cron.d (never looked there)
```
# renew the certificate
3 6 * * * zimbra /opt/letsencrypt-zimbra/obtain-and-deploy-letsencrypt-cert.sh -q
```

there is also a certbot file there:
```
0 */12 * * * root test -x /usr/bin/certbot -a \! -d /run/systemd/system && perl -e 'sleep int(rand(43200))' && certbot -
q renew
```

**Comment by VojtechMyslivec** [comment]
> I changed it's name because of this error:
> ```
> obtain-and-deploy-letsencrypt-cert.sh: warning: You are using deprecated script name, change it to 'letsencrypt-zimbra.sh'
> ```

That means that you call `/.../obtain-and-deploy-letsencrypt-cert.sh` in the crontab (you pasted) instead of `/.../letsencrypt-zimbra.sh` (which is the new name of the script in this repo).


> I just ran it again and it turns out is not working:
> ```
> letsencrypt-zimbra.sh: error: The root CA certificate '/opt/letsencrypt-zimbra/root_certs/ISRG_Root_X1.crt' isn't readable file.
> ```

That means that the script cannot locate that file. Is the `letsencrypt-zimbra` "installed" as described in the [README](https://github.com/VojtechMyslivec/letsencrypt-zimbra#setup-manual)? Maybe you have renamed/moved it to somewhere else. The best way to fix this is probably to drop the `/opt/letsencrypt-zimbra` directory and clone this repo again at `/opt/letsencrypt-zimbra` location. Don't forget to keep your `letsencrypt-zimbra.cfg`.

> I'm so done with zimbra, I will just create a new server with webmin and create a simple mail server, zimbra is way too complicated, etc

I see. I guess you don't need to solve this issue anymore, so I am closing this issue now. Feel free to reopen if you need further assistance.

**Comment by dhcmega** [comment]
It actually worked. I just cloned the repo again, saved my .cfg, and it worked.

I have already created the new server with virtualmin, but I will not shutdown the zimbra server if the cert renewal works in 2 month, so I will wait to see if I switch to virtualmin.

Thank you for taking the time to explain the steps, it only took 2 minutes to clone and generate de certs. I've struggling for 2 weeks now.

dhcmega
